flow-server/src/test/java/com/vaadin/flow/server/frontend/TaskRunPnpmInstallTest.java

```diff
@@ -165,6 +168,7 @@ public void runPnpmInstall_versionsJsonIsFound_pnpmHookFileIsGenerated()
 
         TaskRunNpmInstall task = createTask();
         getNodeUpdater().modified = true;
+        getNodeUpdater().versionsPath = "./version.json";
```

Comment:
Maybe align the filename?

```suggestion
        getNodeUpdater().versionsPath = "./versions.json";
```